docs/home/contribute/style-guide.md

```diff
@@ -144,24 +142,56 @@ The output is similar to this:
     NAME     READY     STATUS    RESTARTS   AGE    IP           NODE
     nginx    1/1       Running   0          13s    10.200.0.4   worker0
 
+### Versioning Kubernetes examples
+
+Code examples and configuration examples that include version information should be consistent with the accompanying text. Identify the Kubernetes version in the **Before you begin** section.
+
+To specify the Kubernetes version for a task or tutorial page:
+
+- Include `min-kubernetes-server-version` in the front matter of the page.
+- In the **Before you begin** section, use `{{ "{% include tasks-tutorial-prereqs.md "}} %}`.
+
+If the example YAML is in a standalone file, find and review the topics that include it as a reference.
+Verify that any topics using the standalone YAML have the appropriate version information defined.
+If a stand-alone YAML file is not referenced from any topics, consider deleting it instead of updating it.
+
+For example, if you are writing a tutorial that is relevant to Kubernetes version 1.8, the front-matter of your markdown file should look something like:
```

Comment:
Random capital L's?
```
For example, in a tutorial for Kubernetes version 1.9, the front matter of the markdown file should look something like:
```
So:
> For example, in a tutorial for Kubernetes version 1.9, the front matter of the markdown file should look something like:

Reply:
I'm a bit clueless about the comment of random capital L's... are you sure it's not something specific about a local font, because those look lowercase to me...